atrain_match/truths/amsr.py

```diff
@@ -32,7 +32,34 @@
 TAI93 = datetime(1993, 1, 1)
 
 logger = logging.getLogger(__name__)
-AMSR_RADIUS = 5.4e3  # 3.7e3 to include 5km pixels parly overlapping amsr-e footprint
+
+
+def calculate_amsr_rof(overlap, imager_radius):
+    """ Calculate radius of influence for nearest neighbour search. 
+        
+        100% overlap:  fov_semi_minor_axis_diam / 2 -IMAGER_RADIUS
+        50% overlap :  fov_semi_minor_axis_diam / 2 
+        25% overlap:  fov_semi_minor_axis_diam / 2  + 0.5*IMAGER_RADIUS
+    """
+    fov_semi_minor_axis_diam = 7000.
+    rof = 0.5*fov_semi_minor_axis_diam + imager_radius - 2*overlap*imager_radius
+    return rof
+
+
+def get_amsr_rof(SETTINGS, imager):
+    """ Get radius of influence as a function of overlap and imager pixel size. """
+    if SETTINGS['AMSR_IMAGER_RADIUS'] < 0:
+        if imager == 'seviri':
+            # pixel size 4.8 x 3 km
+            imager_radius = 5660.
+        elif imager == 'avhrr':
```

Comment:
We also match AVHRR for 1km AVHRR_data

Reply:
Is there an attribute in `cloudproducts` to distinguish them ?

Reply:
Solved with 66bcbbf